docs/guides/container-modules.md

```diff
@@ -31,7 +31,7 @@ kind: Module
 type: container
 name: my-container
 build:
-  depdendencies: [base-image]
+  dependencies: [base-image]
```

Comment:
As this is a code example fixing the typo might break something maybe 🤔.
I found no other mention of `depdendencies` in the code so I doubt it though.